common.gypi

```diff
@@ -1,17 +1,7 @@
 {
   'target_defaults': {
     'default_configuration': 'Release',
-    'cflags_cc' : [
-      '-std=c++14',
-      # The assumption is that projects based on node-cpp-skel will also
-      # depend on mason packages. Currently (this will change in future mason versions)
-      # mason packages default to being built/linked with the CXX11_ABI=0.
-      # So we need to link this way too. This allows source
-      # compiling your module on any ubuntu version, even the latest
-      # where the CXX11_ABI default has flipped to 1
-      # More details at https://github.com/mapbox/mason/issues/319
-      '-D_GLIBCXX_USE_CXX11_ABI=0'
-    ],
```

Comment:
@johnk1 Just looking through this and seeing this flag was removed, and wondering if it seems like it may be correlated?